Puzzlepart/did
# Sticky nav user setting

### Your checklist for this pull request
 
### Review checklist

### Reviewer feedback
- [ ] Tabs are hidden by the sticky navigation
- [ ] Content jumping _(see comment below)_

### Review theme song
_Remove section if you think this is stupid._
🎵  [Chali 2na - Get Involved feat. Defunk](https://open.spotify.com/track/20bXCMaPupQ3ZQlURtRR9D?si=Ec_idmxlTUWlU3_388aGhQ) 🎵 

### Description
Adds a new user setting "Sticky navigation".  
If toggled on, sets <nav> to position: fixed, and <app> with margin-top 65px.  
Allows for per-user setting of sticky nav.

Also makes <nav> full width and avoids the nav "jumping" when scrollbars are rendered.  
Thus, we no longer need `overflow-y: scroll` on the <html>-element in `_template.hbs`

This property should be stored in 
```
[userID] : {
 "configuration" : {
    "ui": { 
      "stickyNavigation": true    
    }
  }
}
``` 

### How to test
1. Check out locally with [gh](https://github.com/cli/cli)
2. log in
3. open user settings
4. toggle "sticky navigation"
5. observe the page reloads
6. navigate to a page with >100% viewport height
7. scroll and observe the nav is sticky
10. repeat step 3-6, and observe the nav is no longer sticky

### Related issues
Closes #852
